Package.swift

```diff
@@ -111,10 +111,10 @@ let package = Package(
             dependencies: []),
         .target(
             name: "WalletConnectVerify",
-            dependencies: ["WalletConnectUtils"]),
+            dependencies: ["WalletConnectUtils", "WalletConnectNetworking"]),
```

Comment:
probably would make sense in the future to extract HTTP from WalletConnectNetworking to not make verify dependent to the whole networking